.github/workflows/tag.yml

```diff
@@ -0,0 +1,61 @@
+name: Tag
+on:
+  workflow_dispatch:
+  workflow_call:
+    inputs:
+      dry-run:
+        description: "Determine the next version without tagging the branch. The workflow can use the outputs new_tag and tag in subsequent steps. Possible values are true and false (default)"
```

Comment:
Did this workflow get copied from BPM? If so, could you move it to https://github.com/broadinstitute/terra-github-workflows instead of duplicating it?

Reply:
Yes I did just copy it over from there.  I agree that terra-github-workflows should be the desired resting place for this GHA logic, but the original author @ichengchang (from whom I copied this) mentioned being on vacation this week and is probably the best person to shepherd (or at least influence) that sort of consolidation.

I see this change as a tactical "fix a broken thing that I broke" + "get this logic some early and additional mileage to inform if any other tweaks are needed before it can be made shareable", which is why it's landing here in pure copypasta format.